main.c

```diff
@@ -631,6 +634,31 @@ static void kbd_keymap(void *data, struct wl_keyboard *k, uint32_t fmt,
 		return;
 	}
 
+	/* Set up XKB compose table */
+	compose_table =
+		xkb_compose_table_new_from_locale(term.xkb_ctx,
+						  getenv("LANG"),
+						  XKB_COMPOSE_COMPILE_NO_FLAGS);
+	if (compose_table) {
+		/* Set up XKB compose state */
+		compose_state = xkb_compose_state_new(compose_table,
+					      XKB_COMPOSE_STATE_NO_FLAGS);
+		if (compose_state) {
+			xkb_compose_state_unref(term.xkb_compose_state);
+			xkb_compose_table_unref(term.xkb_compose_table);
+			term.xkb_compose_state = compose_state;
+			term.xkb_compose_table = compose_table;
+		} else {
+			fprintf(stderr, "could not create XKB compose state.  "
+				"Disabiling compose.\n");
```

Comment:
This says "disabling compose", but I don't see anything which would remove an earlier compose table and state

Also, there is a typo in "Disabling"